libcipher/caesar.py

```diff
@@ -0,0 +1,76 @@
+__author__ = 'kskrishnasangeeth'
+
+import string
+
+
+def cipher_helper(message, key):
```

Comment:
so *dunder* makes it superprivate while single underscore makes it semiprivate

Was wondering the difference between superprivate  and semiprivate in this context.